CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 0.8.0
+
+This version is being released in order to correspond with the state of the proposal that reached Stage 3.
+Expect this polyfill to be replaced by other, more production-ready polyfills.
+
+- All supported calendars are now included in the polyfill. Implementations of some calendars may be slow.
+- The previous behaviour of monkeypatching Temporal.Calendar.from() and Temporal.TimeZone.from() to make custom calendars and time zones globally available is no longer supported.
+- Month codes are now formatted as `"M03"` instead of `"3"`
```

Comment:
Let's help spread awareness of leap months:
```suggestion
- Month codes are now formatted as `"M03"` or (`"M03L"` for a leap month) instead of `"3"`
```